core/hosting/HostWithHostFxr/readme.md

```diff
@@ -29,6 +29,7 @@ Build and Run
 1) In order to build and run, all prerequisites must be installed. The following are also required:
 
     * The C++ compiler (`cl.exe` or `g++`) must be on the path.
+      * On Windows, a [developer command prompt](https://docs.microsoft.com/cpp/build/building-on-the-command-line#developer_command_prompt_shortcuts) should be used.
```

Comment:
```suggestion
      * In Visual Studio on Windows, a [developer command prompt](https://docs.microsoft.com/cpp/build/building-on-the-command-line#developer_command_prompt_shortcuts) should be used.
```

Reply:
@rpetrusha thoughts on something more like the below? `In Visual Studio` makes me think it means using the Visual Studio IDE / UI, but we want the command line.

```suggestion
      * On Windows, a [developer command prompt for Visual Studio](https://docs.microsoft.com/cpp/build/building-on-the-command-line#developer_command_prompt_shortcuts) should be used.
```

Reply:
Sorry, @elinor-fung. I missed your question. It occurs to me that we name it based on the name of the menu selection, which is **Developer command prompt for VS <version>**. Since we'd want to leave out the version, then **Developer command prompt for Visual Studio** sounds good.